sdk/formrecognizer/ai-form-recognizer/swagger/2.1-preview.2.json

```diff
@@ -0,0 +1,1973 @@
+{
```

Comment:
should that file be checked in?

Reply:
It is very slightly different than the one that is currently in PR due to an autorest issue. I mean to turn it into a swagger transform before merging.

Reply:
Resolved in the latest set of commits.